src/html/index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="assets/css/main.css">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div class="header">
+                <img src="./assets/images/logo1.png" alt="Икит Сфу" class="header__logo">
+                <nav class="nav">
+                    <ol class="header__navigation site-navigation">
+                        <li class="site-navigation__el site-navigation--color-secondary site-navigation--size-m">
+                            <a href="#">
+                                Главная
+                            </a>
+                        </li>
+                        <li class="site-navigation__el site-navigation--color-secondary site-navigation--size-m">
+                            Информация
+                        </li>
+                        <li class="site-navigation__el site-navigation--color-secondary site-navigation--size-m">
+                            Новости
+                        </li>
+                        <li class="site-navigation__el site-navigation--color-secondary site-navigation--size-m">
+                            Контакты
+                        </li>
+                    </ol>
+                </nav>
```

Comment:
тут проще описать через один компонент, например
```suggestion
                <nav class="nav header__navigation">
                    <ol class="nav__menu">
                        <li class="nav__el ">
                            <a href="#">
                                Главная
                            </a>
                        </li>
                        <li class="nav__el ">
                            Информация
                        </li>
                        <li class="nav__el ">
                            Новости
                        </li>
                        <li class="nav__el ">
                            Контакты
                        </li>
                    </ol>
                </nav>
```

по поводу модификаторов, тут важно вот что понять, сначала находим базовое состояние и верстаем через элементы, если где то потом нужно отобразить другое состояние, например пункта меню, тогда уже и добавляем модификаторы, например nav__el--size-m nav__el--color-secondary